paperqa/agents/search.py

```diff
@@ -359,18 +358,27 @@ async def maybe_get_manifest(
         try:
             async with await anyio.open_file(filename, mode="r") as file:
                 content = await file.read()
-            records = [DocDetails(**row) for row in csv.DictReader(StringIO(content))]
+            records = [DocDetails(**r) for r in csv.DictReader(content.splitlines())]
+            file_loc_to_records = {
+                str(r.file_location): r for r in records if r.file_location
+            }
+            if not file_loc_to_records:
+                raise ValueError(  # noqa: TRY301
+                    f"No mapping of file location to details extracted from manifest"
+                    f" file {filename}."
+                )
```

Comment:
Without this check, manifest CSV's without a header line would silently pass this